modules/core/src/smithy4s/schema/Schema.scala

```diff
@@ -90,8 +90,11 @@ sealed trait Schema[A]{
   final def biject[B](to: A => B)(from: B => A) : Schema[B] = Schema.bijection(this, to, from)
   final def option: Schema[Option[A]] = Schema.option(this)
 
+  final def nullable: Schema[Nullable[A]] = Nullable.schema(this)
+
   final def isOption: Boolean = this match {
     case _: OptionSchema[_] => true
+    case BijectionSchema(underlying, _) => underlying.isOption // TODO AJ: does this have any side-effects in the other places .isOption is used? is it a breaking change?
```

Comment:
I don't think `.isOption` is used anymore. It was probably used transiently, but I think the change is fine. 

Probably worth adding a `case RefinementSchema` too, just in case.

Reply:
It shows up once in a place involving `flexibleCollectionSupport` in the JSON codec ( https://github.com/disneystreaming/smithy4s/blob/aaf8b1f034b7d3f97e6f250c97fa08c517686d28/modules/json/src/smithy4s/json/internals/SchemaVisitorJCodec.scala#L895 ) but I'm not sure what the logic there is doing.

And good point on the RefinementSchema - will add that!